### ISSUE TITLE
Enhancing data quality in data science and machine learning 

### ISSUE THREAD

**Issue by Doggedben000** [section-engineering-education/engineering-education#4558]
#### Proposed title of article
Enhancing data quality in data science and machine learning 

#### Proposed article introduction
Every data scientist and Machine Learning evangelist's ultimate goal is to build a better model with higher predictive accuracy. However, data may be the issue when it comes to fine-tuning hyperparameters or refining modeling methods. 
No matter how powerful a Machine Learning model is, if the data is typically of poor quality, the results will always be substandard at best. This article seeks to provide ways to enhance data quality for better results.

#### Key takeaways 

the reader should be able to
- understand data and it's role in machine learning and artificial intelligence
- understand the importance of working with quality data
- know some techniques to achieve quality data 

#### Article quality
This article is well elaborated with less technical terms to enable beginners and experts understand ways to enhance data quality in data science and machine learning.
